spec/services/protocol_importers/build_protocol_from_client_service_spec.rb

```diff
@@ -25,13 +25,10 @@
   context 'when have valid arguments' do
     before do
       allow_any_instance_of(ProtocolImporters::ProtocolsIO::V3::ProtocolNormalizer)
-        .to(receive(:load_protocol).and_return(normalized_response))
+        .to(receive(:normalize_protocol).and_return(normalized_response))
       # Do not generate and request real images
       allow(ProtocolImporters::AttachmentsBuilder).to(receive(:generate).and_return([]))
     end
-    it 'Adds Protocol record' do
```

Comment:
Why did you remove this test?

Reply:
It didn't pass, helloo? :D This class only builds the protocol, no longer imports it to DB.

Reply:
I added another one to catch the above bugs.